### ISSUE TITLE
Leave and rename v2

### ISSUE THREAD

**Issue by Mercury13** [notofonts/duployan#6]
Sometimes v2 is needed: it’s smaller, faster and works everywhere. So just leave v2, renaming it (I can’t make a nice name for it).

**Comment by simoncozens** [comment]
Yes, will do this. I've realised that Android will need a "smaller faster" version.

**Comment by dscorbett** [comment]
I recommend subsetting version 3, as in https://github.com/satbyy/go-noto-universal/commit/c6538bd69ca171d526fb1b0b422e1bf0406dde74. That produces a smaller, faster Duployan font that works everywhere, is stylistically consistent with the full font, and has more features than version 2.000. Its name could be “Noto Sans Duployan Unjoined” (cf. Adlam and N’Ko).

**Comment by simoncozens** [comment]
Version 2 is 18704 bytes, version 3 subsetted is 10x larger.

**Comment by dscorbett** [comment]
I’ve improved the subsetting since version 3.001. Now the full Noto font is 1,400,320 bytes and the subset is 84,980 bytes.
